war/src/main/webapp/css/style.css

```diff
@@ -1891,3 +1891,28 @@ body.no-sticker #bottom-sticker {
+
+.remodal {
```

Comment:
I am sorry I don't quite get how should I do it. Since I did not modify form itself, it seems that I only modify nested blocks: https://gist.github.com/samatdav/ca8085e4e117edbc25a010b6174386d1
However, are blocks named any different in bem if there are no elements and modifiers?